scripts/windows-build-fresh-daedalus.bat

```diff
@@ -0,0 +1,29 @@
+rem DEPENDENCIES:
+rem   1. Node.js ('npm' binary in PATH)
+rem   2. 7zip    ('7z'  binary in PATH)
+rem   3. Git     ('git' binary in PATH)
+
+@set DEFAULT_BRANCH=cardano-sl-0.4
```

Comment:
This is different -- this script is supposed to live outside the repository -- since it's responsible for a convenient developer workflow -- shortcutting it allows for a single-click end-to-end fresh installer build, basically.

And since it lives outside the repository.. well, any notion of "current" is removed not by one, but by two steps:
1. finding out the repository location
2. inferring the Cardano branch -- which is a problem of its own

In short, I don't see how we can do much here..